resources/js/Pages/Collections/Components/Articles/ArticlePagination.tsx

```diff
@@ -11,7 +11,7 @@ interface Properties {
 export const ArticlePagination = ({ pagination, onPageLimitChange }: Properties): JSX.Element => {
     const { t } = useTranslation();
 
-    if (pagination.meta.total < 12) {
+    if (pagination.meta.total < pagination.meta.per_page) {
```

Comment:
> We only show this if there is more than 12 articles associated with collection. If less or equal we hide it. By default we have 12 selected on for how many items we show per page. Copy over logic for how we do for NFTs already. Options are 12, 24, 48, 96.

`pagination.meta.total < pagination.meta.per_page` this check will hide the pagination when `pageLimit=24` and `total = 18`